evaluator/opa.go

```diff
@@ -117,8 +120,8 @@ func (o *OpaEvaluator) Eval(ctx context.Context, resourceInfo fetching.ResourceI
 	}
 
 	result, err := o.decision(ctx, OpaInput{
-		Result:         fetcherResult,
-		ActivatedRules: o.activatedRules,
+		Result:    fetcherResult,
+		Benchmark: o.benchmark,
```

Comment:
I am missing some context here, did we change all the rules in a way they will run only when their benchmark is on?